examples/deprecation/deprecation.bal

```diff
@@ -0,0 +1,68 @@
+// The constant `CITY` is marked as deprecated using the `@deprecated` annotation and the markdown
+// documentation `# # Deprecated`.
+# Define a constant to represent the city.
+# # Deprecated
+# This constant is deprecated as this information is not needed.
+@deprecated
+const string CITY = "COLOMBO";
+
+const string COUNTRY = "Sri Lanka";
```

Comment:
These two constant names, they actually sound like variable names. Shall we rename them to something like "CMB" and "LK"? Just so that they sound like constants defined to represent a particular constant value.